packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageSearchBar.tsx

```diff
@@ -48,28 +57,40 @@ export function TracePageSearchBarFn(props: TracePageSearchBarProps & { forwarde
   };
 
   return (
-    <div className="ub-flex-auto ub-mx2 TracePageSearchBar">
+    <div className="TracePageSearchBar">
       {/* style inline because compact overwrites the display */}
-      <Input.Group compact style={{ display: 'flex' }}>
+      <Input.Group className="TracePageSearchBar--InputGroup" compact style={{ display: 'flex' }}>
         <UiFindInput
           inputProps={uiFindInputInputProps}
           forwardedRef={forwardedRef}
           trackFindFunction={trackFilter}
         />
-        <Button
-          className={navigationBtnClass}
-          disabled={navigationBtnDisabled}
-          htmlType="button"
-          icon="up"
-          onClick={prevResult}
-        />
-        <Button
-          className={navigationBtnClass}
-          disabled={navigationBtnDisabled}
-          htmlType="button"
-          icon="down"
-          onClick={nextResult}
-        />
+        {navigable && (
+          <>
+            <Button
+              className={cx(btnClass, 'ant-btn-icon-only')}
+              disabled={!textFilter}
+              htmlType="button"
+              onClick={focusUiFindMatches}
+            >
+              <IoAndroidLocate />
```

Comment:
Does the icon look a bit low?

```css
margin-top: -4px;
```

Reply:
I couldn't tell if it was actually lower or if the up/down arrows are high. when compared to the close icon it seemed closer. I believe it's location is fixed now.